.github/ISSUE_TEMPLATE/theia_update.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Theia CR update
+about: Update Theia to newest community release
+---
+
+<!-- Please replace the version below with the desired version. -->
+# Update Theia to 1.XX.X
+
+## Required Steps
+
+- [ ] Merge latest Theia Blueprint community release commit
+- [ ] if required: Update Theia to latest version of said community release
+- [ ] Update all consumed Theia extensions and VSCode extensions
+- [ ] Test Electron Blueprint locally
+- [ ] Test packaged Electron Blueprint locally
+
+After Merge
+
+- [ ] Download a previous release and check that update to latest version completed successfully
+
+Deploy to try.theia-cloud.io:
+
+- [ ] Push built docker image to registry
+- [ ] Update [deployment file](https://github.com/eclipsesource/theia-cloud/blob/main/demo/k8s/appdefinitions/cdt.yaml) to use newly pushed docker image
+- [ ] Apply the [deployment file](https://github.com/eclipsesource/theia-cloud/blob/main/demo/k8s/appdefinitions/cdt.yaml) to the cluster
```

Comment:
We should use permanent links in case the files are renamed / moved at some point

Reply:
So should i add a permalink to the current state of the file here? I would find that weird if we then point to a really outdated version of the file. Maybe leading to confusion again on what is deployed to the cluster. When this file moves or is renamed i would expect us to update the link here as well. WDYT?